src/datafile.rkt

```diff
@@ -29,6 +29,58 @@
                note
                tests))
 
+;; Calculate the maximum cost and accuracy, initial cost and accuracy, and
```

Comment:
This was just moved? Makes the diff hard to read.

Reply:
Ok, got it. Did you make any changes? (Did you move it in a separate commit from changing it?) I'm just trying to review the changes. If it's not in a commit somewhere, can you describe them in detail?